clippy_lints/src/module_style.rs

```diff
@@ -91,6 +92,13 @@ impl EarlyLintPass for ModStyle {
         let mut file_map = FxHashMap::default();
         for file in files.iter() {
             if let FileName::Real(name) = &file.name && let Some(lp) = name.local_path() {
+                if file.cnum != LOCAL_CRATE {
```

Comment:
```suggestion
            if let FileName::Real(name) = &file.name
                && let Some(lp) = name.local_path()
                && if file.cnum == LOCAL_CRATE
            {
```
Let's combine these two if-statements, since it seems `continue`-ing is unnecessary